### ISSUE TITLE
User identifier tooltip broken on IE11

### ISSUE THREAD

**Issue by mrpau-richard** [learningequality/kolibri#6336]
### Observed behavior
User identifier tooltip not showing the full text message on IE11

### Expected behavior
Should display the text message properly

### User-facing consequences

…

### Errors and logs
![Screen Shot 2019-12-16 at 12 47 52 PM](https://user-images.githubusercontent.com/8663934/70880173-f9ba9700-2002-11ea-86cb-6aa85f2a7990.png)

Please wrap errors in triple backticks for clean formatting like this:
```
01:10 info: something happened
01:12 error: something bad happened
```
-->

…

### Steps to reproduce

…

Tell us about your environment, including:
 * Kolibri version: 0.13.0b7
 * Operating system: Windows 7
 * Browser: IE11

**Comment by indirectlylit** [comment]
refs:

* https://github.com/learningequality/kolibri/issues/5106
* https://github.com/learningequality/kolibri/issues/6069

**Comment by pcenov** [comment]
@radinamatic this one is fixed in 0.15.6:
![2022-08-03_15-32-44](https://user-images.githubusercontent.com/79847249/182609180-27ebe66a-23ed-417e-b39e-7403616814a3.png)

**Comment by rtibbles** [comment]
Closing as fixed.
